compiler/typecheck/step3_function_and_method_bodies.jou

```diff
@@ -671,7 +790,12 @@ def typecheck_class_field(
         return f
 
     msg: byte[500]
-    snprintf(msg, sizeof(msg), "class %s has no field named '%s'", classtype.name, fieldname)
+
+    if classtype.find_method(fieldname) == NULL:
+        snprintf(msg, sizeof(msg), "class %s has no field named '%s'", classtype.name, fieldname)
+    else:
+        snprintf(msg, sizeof(msg), "%s.%s() is a method, it can be called but it cannot be used in any other way", classtype.name, fieldname)
```

Comment:
Should perhaps write a test that tries to capture a method into a variable. (This error message already appears in a test that tries to assign to a method.)